src/components/UsersCard.js

```diff
@@ -69,18 +77,51 @@ function UserRow({ model }) {
         </Form.Control>
       </td>
       <td>{model.email}</td>
+      <td>
+        <InstructorSelect
+          defaultValue={instructor}
+          onChange={(event) => setInstructor(event.target.value || null)}
+        />
+      </td>
       <td>{formatTimestamp(model.createdAt)}</td>
       <td className="text-right">
         <Button
-          onClick={changeAccessLevel}
+          onClick={changeAccess}
           size="sm"
           variant="primary"
           className="ml-3"
-          disabled={accessLevel === model.accessLevel}
+          disabled={!accessLevelChanged && !instructorChanged}
```

Comment:
I was able to give a user with `accessLevel` = `revoked` access to an instructor, which doesn't seem ideal.

Maybe that can be solved by adding `isRevoked` to this button too?

Reply:
Updated to disable instructor selection when the access-level is revoked 👍